sdk/nodejs/appservice/zMixins.ts

```diff
@@ -838,7 +838,7 @@ declare module "./functionApp" {
         /**
          * Retrieve the keys associated with the Function App.
          */
-        getHostKeys(): pulumi.Output<FunctionHostKeys>;
+        getHostKeys(attempts?: number): pulumi.Output<FunctionHostKeys>;
```

Comment:
I'd prefer not to change the signature of this function and just always have `5` hard-coded for now.

Reply:
Have removed the parametrisation 😊